cypress/e2e/item/thumbnails/viewThumbnails.cy.js

```diff
@@ -29,10 +26,9 @@ describe('View Thumbnails', () => {
 
     // GRID
     cy.switchMode(ITEM_LAYOUT_MODES.GRID);
-    cy.wait(TABLE_ITEM_RENDER_TIME);
     cy.get(`#${buildItemCard(items[0].id)} img`)
       .should('have.attr', 'src')
-      .and('contain', DEFAULT_IMAGE_SRC);
+      .and('contain', 'data:');
```

Comment:
Should we use the exact base64 data?

Reply:
Mhmm we "should" but this requires to add a loader to cypresse's webpack-config, if I am not mistaken, and I don't know how to do that.

Reply:
Or we just copy paste it from the browser, which is a bit ugly no ?

Reply:
I think it could make sense. In general I think using raw data in tests in general is a good practice (you don't want logic errors that could cause bugs in the tests themselves). However I do agree that the way the image is bundled is obscure (the compression may change, and depending on the size several sizes may be generated), so it could change while out of our influence. Not sure what's best.